libraries/rush-lib/src/logic/base/BaseLinkManager.ts

```diff
@@ -80,6 +81,15 @@ export abstract class BaseLinkManager {
     }
   }
 
+  public static _symlinkExists(linkPath: string): boolean {
```

Comment:
Why is this public but prefixed with `_`?  Function names should start with a verb.

Suggest to move this to [Utilities.ts](https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/utilities/Utilities.ts), use `FileSystem.getLinkStatistics()` instead of `lstatSync()`, and add a `/** */` more clearly explaining the purpose of this function:
```ts
/** 
 * If the path refers to a symlink, `FileSystem.exists()` would normally test whether the symlink
 * points to a target that exists.  By contrast, `existsOrIsBrokenSymlink()` will return true even if
 * the symlink exists but its target does not.
 */ 
  public static existsOrIsSymlink(path: string): boolean {
  ```

Reply:
changed